src/server/html.js

```diff
@@ -11,9 +11,9 @@ export default function (body, initialState, path) {
     </head>
     <body>
       <div id="mount">${body}</div>
-      <script id="initialState" type="application/json">${JSON.stringify(initialState)}</script>
-      <script type="text/javascript" src="${path}/index.js"></script>
+      ${includeJS ? '<script id="initialState" type="application/json">' + JSON.stringify(initialState) + '</script>' +
```

Comment:
Should we move this above the return? Could split into two const's, state and script?